qxf2trainer/templates/vscode_setup/tutorial.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'base.html' %} {% block title %}Qxf2 Trainer: Tutorial{% endblock %} {% block content %}
+
+<p class="text-justify">We expect this exercise to take you somewhere between 15-minutes to 2 hours. If you take more than two hours, please post what you tried along wtih error messages to our slack channel.</p>
+
+<p class="text-justify">In keeping with our attempt to help you think in different ways, we have listed three approaches that work. There is no shame or pride in choosing anyone of them. Just choose the method you are comfortable with and proceed.</p>
+<p>&nbsp;</p>
+<h3 class="thin-text">
+    <img src="https://qxf2.com/assets/img/approach_icon.png" width="60" height="60"> Approach 1: Try it out yourself
+</h3>
+<ol>
+    <li>We'd like to promote independent thinking and problem solving. So if you feel ready, try to install Python without referring to any documentation, go for it!</li>
+</ol>
+<p>&nbsp;</p>
+<h3 class="thin-text">
+    <img src="https://qxf2.com/assets/img/approach_icon.png" width="60" height="60"> Approach 2: Google for instructions
+</h3>
+<ol>
+    <li>Google for visual studio code install 'your operating system name'
+    </li>
+    <li>Follow the instructions</li>
+</ol>
+<p>&nbsp;</p>
+<h3 class="thin-text">
+    <img src="https://qxf2.com/assets/img/approach_icon.png" width="60" height="60"> Approach 3: follow instructions
+</h3>
+<ol>
+    <li>Visit <a href="https://code.visualstudio.com/download">https://code.visualstudio.com/download</a> to install visual studio code for your operating system</li>
+    <li>Visit <a href="https://code.visualstudio.com/download">https://code.visualstudio.com/download</a> to install python extension for vscode</li>
+    <li>Start python coding on vscode</li>
+    <li>Vist <a href="https://marketplace.visualstudio.com/items?itemName=ms-python.python">https://marketplace.visualstudio.com/items?itemName=ms-python.python</a> help documentation for any questions</li>
```

Comment:
@rajiqxf2 - we should never ever point our users to a help doc like this. It becomes a black hole for them and they go back to thinking like they previously did - "Oh, I need to read a lot before I try". At best, we can point to specific sections in help doc but we should *always* preface that by telling them NOT to read the whole thing. Just read the snippet they want and come out.